shared/desktop/app/app-state.js

```diff
@@ -199,6 +201,9 @@ export default class AppState {
       const stateLoaded = JSON.parse(fs.readFileSync(configPath, {encoding: 'utf8'}))
       if (this._isValidState(stateLoaded)) {
         this.state = stateLoaded
+      } else {
```

Comment:
we actually want all the rest of the state and really just ignore the window stuff so i'd argue the flow should be
```
if (!this._isValidState(stateLoaded)) {
      stateLoaded.x = null
      stateLoaded.y =  null
}

this.state = stateLoaded
```

Reply:
changed.

Reply:
no i mean , we still want to load state, we just don't want to keep the x/y if the screen dimensions changed, but all other fields should be loaded

Reply:
See `dec5db9`